examples/atari/train_double_iqn.py

```diff
@@ -140,7 +143,15 @@ def make_env(test):
     opt = chainer.optimizers.Adam(5e-5, eps=1e-2 / args.batch_size)
     opt.setup(q_func)
 
-    rbuf = replay_buffer.ReplayBuffer(10 ** 6)
+    if args.prioritized:
+        betasteps = args.steps / args.update_interval
+        rbuf = replay_buffer.PrioritizedReplayBuffer(
+            10 ** 6, alpha=0.5, beta0=0.4, betasteps=betasteps,
```

Comment:
I didn't find `alpha` for prioritized replay in the paper https://arxiv.org/abs/1710.10044.  Could you confirm where the hyperparameters came from?

Reply:
The hyperparameters don't come from anywhere. Prioritized Double IQN is a new algorithm.

Reply:
However, I could use `alpha=0.2` and `beta0=0.4`, which appears to be the case here: https://github.com/valeoai/rainbow-iqn-apex/blob/master/rainbowiqn/args.py, which implements Rainbow IQN.